src/components/Discussion/Internal/Comment/Actions.js

```diff
@@ -253,7 +242,12 @@ export const Actions = ({
               actions.featureComment && (() => actions.featureComment(comment))
             }
           >
-            <FeaturedIcon fill={featuredText ? colors.primary : undefined} />
+            <FeaturedIcon
+              {...colorScheme.set(
+                'fill',
+                featuredText ? 'primary' : 'disabled'
```

Comment:
```suggestion
                featuredText ? 'primary' : 'text'
```
because when it's not yet featured admin can click it to feature. Regular don't see it unless featured.